mandatory/2-function-creation.js

```diff
@@ -6,7 +6,10 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+
+function validate(num) {
+  return typeof num==="number" && num%2===0 && num<=100 ? true : false;
```

Comment:
You don't need to include a ternary operator if you are just returning true or false 
`return typeof num==="number" && num%2===0 && num<=100` is equivalent